torch/__init__.py

```diff
@@ -2037,5 +2037,15 @@ def _constrain_as_size(symbol, min: Optional[builtins.int] = None, max: Optional
     torch.sym_constrain_range_for_size(symbol, min=min, max=max)
 
 
+# TODO: what would be a good place for this?
```

Comment:
We usually either inline layout checks or introduce an internal `set` variable containing sparse layouts.

Reply:
removed and replaced with the suggested inline check